apiserver/facades/client/spaces/spaces_test.go

```diff
@@ -1030,12 +1018,13 @@ func (s *LegacySuite) TestListSpacesDefault(c *gc.C) {
 
 func (s *LegacySuite) TestListSpacesAllSpacesError(c *gc.C) {
 	defer s.setupMocks(c).Finish()
+	s.providerGetter = func(stdcontext.Context) (environs.NetworkingEnviron, error) {
+		return nil, errors.New("backing boom")
```

Comment:
I think we should update the text of the error. Only because backing was the name given to the get env stuff we were using and it would be nice to not have the association.